### PR TITLE
Module 3

### PR DESCRIPTION
**Pull request by Bedkow**
tasks 1, 2, 3 done

### CODE REVIEW

File: module-3/task3.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+		noheader: true,
+		headers: ["Book", "Author", "Amount", "Price"],
+		output: "json",
+		downstreamFormat: "line",
+	})
+		.fromString(line)
+		.then((row) => {
+			if (lineNumber > 1) {
+				fs.appendFile(
+					"./txtdirectory/nodejs-m3.t3.txt",
```
Comment (eduardhyan) [inline module-3/task3.js:23]: When I first ran it, it failed. The issue was that "txtdirectory" wasn't included in the git repository. After cloning the repo, it wasn't automatically generated, so I had to create it manually. Kindly implement conditions to ensure that if the directory doesn't exist, the program will create it.

**Review by eduardhyan — CHANGES_REQUESTED** [review]

File: module-3/task1.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+	}
+
+	off(eventName, fn) {
+		return this.removeListener(eventName, fn);
+	}
+
+	removeListener(eventName, fn) {
+		let lis = this.listeners[eventName];
+		if (!lis) return this;
+		for (let i = lis.length; i > 0; i--) {
```
[the remark this answers is not in the document]
Comment (Bedkow) [reply module-3/task1.js]: For loop counting modified according to comment. Thanks!

File: module-3/task3.js (hunk 1 above)
Comment (Bedkow) [reply module-3/task3.js:23]: @eduardhyan the Node.js docs state that if the file does not exist, it gets created with the desired content. Please take a look:
https://nodejs.org/api/fs.html#fsappendfilepath-data-options-callback

Comment (Bedkow) [reply module-3/task3.js:23]: directory added with .gitkeep

**Review by eduardhyan — APPROVED** [review]

**Pull request merged**